docs/how-to-guides/feathr-udfs.md

```diff
@@ -0,0 +1,182 @@
+---
+layout: default
+title: Feathr User Defined Functions (UDFs)
+parent: How-to Guides
+---
+
+# Feathr User Defined Functions (UDFs)
+
+Feathr supports a wide range of user defined functions (UDFs) to allow flexible way of dealing with your data. Feathr supports two use cases:
+
+1. [User defined functions at input sources (also known as preprocessing functions)](#user-defined-functions-at-input-sources-also-known-as-preprocessing-functions)
```

Comment:
Given Users already know `UDFs` = User Defined Functions, we could use abbreviations here and below.